packages/client/components/ScopePhaseArea.tsx

```diff
@@ -108,6 +109,39 @@ const ScopePhaseArea = (props: Props) => {
     setActiveIdx(2)
   }
 
+  // swipeable views won't ignore null children so conditionally create them: https://github.com/oliviertassinari/react-swipeable-views/issues/271
+  const contents = [
```

Comment:
-1 I think this is the right approach, but I don't like that the order here needs to manually be matched to the order of `baseTabs`. Can you refactor it that we have something like:
```ts
const contents: Record<typeof baseTabs[number]['label'], JSX.Element> = {
  'GitHub': <ScopePhaseAreaGitHub
      isActive={isTabActive('GitHub')}
      gotoParabol={goToParabol}
      meetingRef={meeting}
    />,
  'Jira': <ScopePhaseAreaJira
      isActive={isTabActive('Jira')}
      gotoParabol={goToParabol}
      meeting={meeting}
    />,
  'Parabol': <ScopePhaseAreaParabolScoping isActive={isTabActive('Parabol')} meeting={meeting} />,
    'Jira Server': <ScopePhaseAreaJiraServer
      isActive={isTabActive('Jira Server')}
      gotoParabol={goToParabol}
      meetingRef={meeting}
    />,
  'GitLab': <ScopePhaseAreaGitLab
        isActive={isTabActive('GitLab')}
        gotoParabol={goToParabol}
        meetingRef={meeting}
      />
  }
```
and then later iterate over `tabs` to insert the correct component? This way we cannot mess up the order